habitat-lab/habitat/config/habitat/dataset/rearrangement/floorplanner.yaml

```diff
@@ -0,0 +1,12 @@
+# @package habitat.dataset
```

Comment:
I think we can get rid of this now that we have the new `hssd.yaml`?

Reply:
Done

Reply:
Looks like this is back

Reply:
I removed this in the other PR actually. First that needs to be merged here

Reply:
I did not test this independently

Reply:
Deleted here as well